pkgs/development/python-modules/openstep-parser/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "openstep_parser";
```

This should be normalized.

```suggestion
  pname = "openstep-parser";
```